pkg/security/probe/probe_ebpfless.go

```diff
@@ -470,28 +480,40 @@ func (p *EBPFLessProbe) Start() error {
 
 	ch := make(chan clientMsg, 100)
 
+	p.wg.Add(1)
 	go func() {
+		defer p.wg.Done()
+
 		for {
 			conn, err := listener.Accept()
 			if err != nil {
-				seclog.Errorf("unable to accept new connection")
-				continue
+				return
```

Comment:
Should we keep logging errors that are unrelated with the listener being closed?